scripts/release_ppa.sh

```diff
@@ -166,6 +166,9 @@ override_dh_auto_test:
 
 override_dh_shlibdeps:
 	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
+
+override_dh_auto_configure:
```

Comment:
@axic If `INSTALL_LLLC` is off by default then this can be removed.

Reply:
Yes, but kept it there for safety reasons. We may switch it on or who knows :)